nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss

```diff
@@ -334,6 +335,9 @@ $appFontPath: '~roboto-fontface/fonts';
     .refresh-timestamp {
         color: $warn-palette-A400;
     }
+    .skeleton-loader{
```

Comment:
Please add a blank line between `.refresh-timestamp` and `.skeleton-loader` class rules.